src/desktop/chat/useritemdelegate.cpp

```diff
@@ -46,6 +46,7 @@ UserItemDelegate::UserItemDelegate(QObject *parent)
 
 	m_userMenu->addSeparator();
 	m_chatAction = m_userMenu->addAction(tr("Private Message"));
+	m_infoAction = m_userMenu->addAction(tr("Show User Information"));
```

Comment:
This should probably be something that is pushed by the user sharing the information rather than something that is pulled by a user that wants to see it.

Reply:
Would go against the use case of the feature described above.